tests/bootstrap_redis.sh

```
@@ -0,0 +1,110 @@
+        echo "invalid USE_SSL ${USE_SSL} given"
+        exit 1
+    ;;
+esac
+
+case "${NODE_TYPE}" in
+    "standalone")
+        hostname=redis
+        get_server_args 'server_args' "${hostname}"
+        container_name="${CONTAINER_PREFIX}standalone"
+        docker run --name "${container_name}" -v "${SSL_PATH}:/ssl-certs" --rm -d \
+            "${NODE_IMAGE}" ${server_args[@]}
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)